carmin-server/server/resources/decorators/__init__.py

```diff
@@ -65,15 +69,24 @@ def wrapper(*args, **kwargs):
     return decorator
 
 
-def login_required(func):
+def get_db_session(func):
+    @wraps(func)
+    def wrapper(*args, **kwargs):
+        return func(db_session=db.create_scoped_session(), *args, **kwargs)
+
+    return wrapper
+
+
+@get_db_session
```

Comment:
Discussed and agreed to keep it for the moment. In case we feel like needing extra decorators in the future, we will reconsider its importance further and possibly move it to a utils fonction.